internal/stage7.go

```diff
@@ -47,8 +41,20 @@ func testType2(stageHarness *test_case_harness.TestCaseHarness) error {
 
 	for _, executable := range availableExecutables {
 		command := fmt.Sprintf("type %s", executable)
-		actualPath := getPath(executable)
-		expectedPattern := fmt.Sprintf(`^(%s is )?%s\r\n`, executable, actualPath)
+
+		var commandOutput string
+		if executable == "my_exe" {
+			commandOutput = customExecutablePath
+		} else {
+			path, err := exec.LookPath(executable)
+			if err != nil {
+				commandOutput = fmt.Sprintf(`%s[:]? not found`, executable)
```

Comment:
@glokta1 btw, for context - one pattern we use is to distinguish between "user error" and "codecrafters error". When there's an error that must be our fault, we always log it with "CodeCrafters internal error", and that sends us slack alerts in #exceptions. 

Example: https://github.com/codecrafters-io/shell-tester/blob/72976f6dc32633b863ef005937be3d3f3199e8b9/internal/utils.go#L28

This seems like one of those cases. cc: @ryan-gang, in case we had a reason to not make this an internal error?

Reply:
Previously we were using this `getPath()` function on both available and non-available executables. 
The error string returned from `getPath()` is the output we expect from shell. 

As the environments are same, if `lookPath` fails, we expect shell to fail in the same way, so we are not treating this as an internal error.

Reply:
Ah, cool. Summarizing: Looks like we've now separated handling available and non-available executable handling, so if we hit this for a "available" executable, should be an internal error